uspace/lib/c/generic/cap.c

```diff
@@ -159,7 +159,9 @@ void cap_simplify(cap_spec_t *cap)
 	}
 
 	/* Round the number so that we have at most @c scap_max_sdig significant digits */
-	sdig = 1 + ilog10_u64(cap->m); /* number of significant digits */
+	rc = ilog10_u64(cap->m, &sdig); /* number of significant digits */
+	sdig += 1;
+	assert(rc == EOK);
```

Comment:
I don't think we can assume that `cap->m` is never zero.

Reply:
Your right again. But this means I also forgot to implement a test for caps where 0 gets passed. If not I should have noticed it, since it always should fail when `cap->m == 0`. I assume you agree that `ilog10_u64(0)` should indeed fail with an errno?